tests/furusiya/model/skills/test_omnislash.py

```diff
@@ -0,0 +1,12 @@
+from unittest.mock import Mock
+
+from model.skills.omnislash import OmniSlash
+
+
+def test_process():
+    player = Mock()
+
+    for _ in range(100):
+        OmniSlash.process(player, 10, (1, 0))
+
+    assert 20 > player.move_or_attack.call_count > 5  # Roughly estimate. Yay randomness!
```

Comment:
If you want to be a tricksy little hobbits, you can always seed the random generator right before you do this. I don't think that's a valuable thing to add though.